src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -198,7 +196,7 @@ internal void EnqueueBindToDataSource(MirrorData mirrorData)
 
         #region Public Class Properties
 
-        internal bool IsDataBound { get { return mirrorData != null; } }
```

Comment:
Do you expect someone else set this property? If not then I'll suggest making the setter protected or private.